src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesScaleDenominatorTest.java

```diff
@@ -0,0 +1,237 @@
+/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2013 OpenPlans
```

Comment:
Is this a new file? Or are you reusing existing code (and thus need the open plans reference)